--- a/tests/model/test_tablero.py
+++ b/tests/model/test_tablero.py
@@ -11,7 +11,6 @@
 from model.piezas.caballo import Caballo
 from model.piezas.peon import Peon
 from typing import Tuple, Optional, Type
-from model.validador_movimiento import ValidadorMovimiento
 
 # ============================================================
 # Fixtures de Pytest
@@ -168,15 +167,6 @@ def test_getPieza_invalida(tablero_inicial: Tablero):
     assert tablero_inicial.getPieza((-1, 5)) is None
     assert tablero_inicial.getPieza((4, 8)) is None
 
-def test_esBlanco(tablero_inicial: Tablero):
-    """
-    Verifica la identificación correcta del color blanco usando ValidadorMovimiento.
-    """
-    validador = ValidadorMovimiento(tablero_inicial)
-    
-    assert validador.esBlanco((0, 0)) is True # Torre blanca
-    assert validador.esBlanco((1, 4)) is True # Peón blanco
-    assert validador.esBlanco((7, 7)) is False # Torre negra
-    assert validador.esBlanco((6, 2)) is False # Peón negro
-    assert validador.esBlanco((4, 4)) is False # Casilla vacía
-
 # ============================================================
 # Pruebas de Movimiento Básico (moverPieza)
 # ============================================================ 